rel/overlay/etc/default.ini

```diff
@@ -187,6 +187,12 @@ credentials = false
 ; List of accepted methods
 ; methods =
 
+[exclude_headers]
+; List of headers that will be kept when the header option X-Couch-Exclude-Headers is included in a request
+; 'X-Couch-Exclude-Headers' : "All"
+all = Server
+minimal = Cache-Control, Content-Length, Content-Type, ETag, Server, Vary
```

Comment:
Good point. I've added Content-Length, Content-Type and Transfer-Encoding to the all list